translator/combine_wzdx.py

```diff
@@ -55,9 +41,30 @@ def main():
     if combined_wzdx:
         with open(outputfile, 'w+') as f:
             f.write(json.dumps(combined_wzdx, indent=2))
-        print(f'Combined WZDx message was written to {outputfile}.')
+        logging.info(f'Combined WZDx message was written to {outputfile}.')
     else:
-        print('No duplicate WZDx messages were found. Output file was not created')
+        logging.warning(
+            'No duplicate WZDx messages were found. Output file was not created')
+
+
+# parse combination script command line arguments
+def parse_combined_arguments() -> tuple:
+    """Parse command line arguments for combination script
+
+    Returns: 
+        Tuple of (iCone file path, cotrip file path, output file path)
+    """
+    parser = argparse.ArgumentParser(
+        description='Detect and combine duplicate iCone and COTrip WZDx work zone messages')
+    parser.add_argument('--version', action='version',
+                        version=f'{PROGRAM_NAME} {PROGRAM_VERSION}')
+    parser.add_argument('iconeFile', help='icone file path')
+    parser.add_argument('cotripFile', help='cotrip file path')
+    parser.add_argument('--outputFile', required=False,
+                        default='combined_wzdx_message.geojson', help='WZDx output file path')
+
+    args = parser.parse_args()
+    return args.iconeFile, args.cotripFile, args.outputFile
 
 
 def find_duplicate_features_and_combine(wzdx_source: dict, wzdx_destination: dict) -> dict:
```

Comment:
Can we remove "duplicate" and add something more descriptive about what we're actually doing? Maybe "find_overlapping_features_and_combine"?

Reply:
Renamed to find_overlapping_features_and_combine, as well as updating comments/descriptions accordingly